modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -151,4 +154,87 @@ test.describe('Manage object entries through page templates', () => {
 			objectDefinition.id
 		);
 	});
+
+	test('can view all entries related to an object in the relationship field', async ({
+		apiHelpers,
+        page,
+        viewObjectEntriesPage
+	}) => {
+		const objectDefinition1 =
+        await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			objectFolderExternalReferenceCode: 'default',
+			status: {code: 0},
+		});
+
+		const objectDefinition2 =
+        await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			objectFolderExternalReferenceCode: 'default',
+			status: {code: 0},
+		});
+
+        const objectRelationshipLabel =
+			'objectRelationshipLabel' + getRandomInt();
+		const objectRelationshipName =
+			'objectRelationshipName' + Math.floor(Math.random() * 99);
+
+        const objectRelationshipData: Partial<ObjectRelationship> = {
+			label: {
+				en_US: objectRelationshipLabel,
+			},
+			name: objectRelationshipName,
+			objectDefinitionExternalReferenceCode1:
+				objectDefinition1.externalReferenceCode,
+			objectDefinitionExternalReferenceCode2:
+				objectDefinition2.externalReferenceCode,
+			objectDefinitionId1: objectDefinition1.id,
+			objectDefinitionId2: objectDefinition2.id,
+			objectDefinitionName2: objectDefinition2.name,
+			type: 'oneToMany' as ObjectRelationshipType,
+		};
+
+		await apiHelpers.objectAdmin.postObjectRelationship(
+			objectRelationshipData
+		);
+
+        const applicationName =
+        'c/' + objectDefinition1.name.toLowerCase() + 's';
+
+        const textObjectEntry = {
+            textField: 'entry',
+        };
+
+        for (let i = 0; i <= 15; i++) {
+            await apiHelpers.objectEntry.postObjectEntry(
+                textObjectEntry,
+                applicationName
+            );
+        }
+
+        await viewObjectEntriesPage.goto(objectDefinition2.id);
+        await viewObjectEntriesPage.addObjectEntryButton.click();
+        await page.getByPlaceholder('Search', { exact: true }).click();
+
+        const objectEntries = await apiHelpers.objectEntry.getObjectDefinitionObjectEntries(applicationName);
+        const totalCount = objectEntries.totalCount;
+
+        const objectEntriesList = await page.locator(
+             '.autocomplete-dropdown-menu.show > div > ul.list-unstyled li'
```

Comment:
Hey @vngoliveira could you check if this locator could replace the use of this css selector?

`page.getByLabel('textObjectEntry.textField');`

Thanks 😄

Reply:
Furthermore, remember to make the changes suggested by the SF: 

>      [exec] tests/object-web/objectEntries.spec.ts(161,9): error TS2339: Property 'viewObjectEntriesPage' does not exist on type 'PlaywrightTestArgs & PlaywrightTestOptions & \{ apiHelpers: ApiHelpers; \} & \{ collectionsPage: CollectionsPage; \} & ... 7 more ... & PlaywrightWorkerOptions'.
>      [exec] tests/object-web/objectEntries.spec.ts(207,30): error TS2339: Property 'objectEntry' does not exist on type 'ApiHelpers'.
>      [exec] tests/object-web/objectEntries.spec.ts(217,48): error TS2339: Property 'objectEntry' does not exist on type 'ApiHelpers'.

Reply:
Hmm, I see, to fix these flaws if SF pointed out. You need the changes that are in Daniel's PR. Correct?

Reply:
Yes, to send this PR to Brian, I will need to wait for Daniel's PR.